### PR TITLE
feat(admin-api): Add field resolver for allowed scopes in client

### PR DESCRIPTION
**Pull request by snaerseljan**
# Add field resolver for allowed scopes in client

- Added field resolver for allowedScopes to getClientById in resolver
- Added ClientAllowedScopesDataLoader to get clients allowed scopes
- Added Rest endpoint to fetch allowed scopes by client and tenant id.

TODO

- [x] Create Test for rest endpoint.

## Checklist:

- [x] I have performed a self-review of my own code
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] Formatting passes locally with my changes
- [x] I have rebased against main before asking for a review

### CODE REVIEW

**Review by saevarma — APPROVED** [review]
💯
Just few nit picks.

File: libs/auth-api-lib/src/lib/resources/admin/admin-scope.service.ts [hunk 1]
```diff
@@ -31,4 +38,52 @@ export class AdminScopeService {
 
     return apiScopes.map(this.mapApiScopesToDto)
   }
+
+  async findAllowedScopes({
```
[the remark this answers is not in the document]
Comment (snaerseljan) [reply libs/auth-api-lib/src/lib/resources/admin/admin-scope.service.ts]: Good suggestions. I have updated.

File: libs/api/domains/auth-admin/src/lib/client/clients.service.ts [hunk 2]
```diff
@@ -15,10 +15,11 @@ import { CreateClientInput } from './dto/create-client.input'
 import { CreateClientResponse } from './dto/create-client.response'
 import { PatchClientInput } from './dto/patch-client.input'
 import { PublishClientInput } from './dto/publish-client.input'
-import { AllowedScopeInput } from './dto/allowed-scope.input'
+import { ClientAllowedScopeInput } from './dto/client-allowed-scope.input'
 import { ClientEnvironment } from './models/client-environment.model'
 import { ClientSecret } from './models/client-secret.model'
 import { Client } from './models/client.model'
+import { MeClientsScopesController } from '../../../../../../../apps/services/auth/admin-api/src/app/v2/scopes/me-clients-scopes.controller'
```
Comment (saevarma) [inline libs/api/domains/auth-admin/src/lib/client/clients.service.ts:22]: This import shouldn´t be here

**Review by saevarma — APPROVED** [review]
💯

**Pull request merged**